### DISCUSSION TITLE
[Funcionamento na versão 1.1]

### DISCUSSION THREAD

**Discussion by diogosm** [qualitaocupacional/libesocial#9] [General]
Alguém obteve sucesso usando a lib com leiaute 1.1?

Tive êxito recentemente usando na 1.0, mas com o gov lançou uma nota sobre o fim desse leiaute. Tentei executar usando o novo leiaute, mas tive erros, alguém já passou por isso aqui?

Ref:
[Fim da convivencia das versoes de leiaute](https://www.gov.br/esocial/pt-br/noticias/fim-da-convivencia-das-versoes-do-leiaute-do-esocial)

**Comment by diogosm** [comment]
Acabei tentando mudar a variável global no xml.py via classe do meu código, mas não deu certo (com a lib do PyPi). Só funcionou quando baixei o projeto, na branch develop (vi q tinha um commit pra isso) e instalei direto a lib. Obrigado gente, essa lib é fantástica!

**Reply by andidevel** [reply]
Olá @diogosm, obrigado pelo feedback e por usar a LIBeSocial. Fico feliz em saber que esta te ajudando de alguma forma.

Realmente a branch develop está com "mais features" do que a main, e acabamos não criando uma nova release no PyPi. Estamos fazendo um "grande" refactor para facilitar a atualização de futuros novos layouts, 😃 .
